src/DetailsView/reports/components/outcome-type.ts

```diff
@@ -4,16 +4,16 @@ import { countBy, values } from 'lodash';
 
 import { ManualTestStatus, ManualTestStatusData } from '../../../common/types/manual-test-status';
 
-export type OutcomeType = 'pass' | 'incomplete' | 'fail';
+export type RequirementOutcomeType = 'pass' | 'incomplete' | 'fail';
 
-export const allOutcomeTypes: OutcomeType[] = ['pass', 'incomplete', 'fail'];
+export const allOutcomeTypes: RequirementOutcomeType[] = ['pass', 'incomplete', 'fail'];
```

Comment:
Consider renaming this to `allRequirementOutcomeTypes` to make clear that it doesn't contain `'inapplicable'`.